jitsi-media-transform/src/main/kotlin/org/jitsi/nlj/PacketInfo.kt

```diff
@@ -56,7 +58,7 @@ class EventTimeline(
     }
 
     fun clone(): EventTimeline {
-        val clone = EventTimeline(ConcurrentLinkedQueue(timeline))
+        val clone = EventTimeline(timeline.toMutableList())
```

Comment:
Not a new bug, but while you're in there, `clone()` should pass `clock` rather than letting it default to `Clock.systemUTC()`.